src/api/ApiIndex.vue

```diff
@@ -51,9 +51,9 @@ const filtered = computed(() => {
 <template>
   <div id="api-index">
     <div class="header">
-      <h1>API Reference</h1>
+      <h1>API довідник</h1>
```

Comment:
мені здається, що так буде краще
```suggestion
      <h1>Довідник API</h1>
```